server/routes/loginAuth.js

```diff
@@ -5,7 +5,8 @@ const dayjs = require("dayjs");
 const logger = require("../logger");
 
 require("./loginStrategies/googleStrategy");
```

Comment:
Also make, google optional, for the instance where the facebook config is provided instead of google. Then later when you are checking which services are configured, crash the app if the none at all are provided